validator/testdata/amp4ads_feature_tests/amp_selector.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
```

Comment:
please add the license file here

Reply:
Done. Also updated the list in extensions.html.

Reply:
Forgot about this test for allowed extensions in AMP Ads: https://github.com/ampproject/amphtml/blob/master/validator/engine/validator_test.js#L1265

Reply:
Done. Thanks!